.github/renovate.json5

```
@@ -75,9 +89,14 @@
       "enabled": false
     },
     {
-      // Ignore dependency updates from github.com/gardener because these PRs are created by pipeline jobs.
-      "matchDatasources": ["go"],
-      "matchPackagePatterns": ["github\\.com\/gardener\/.+"],
+      // Update only patchlevels container images of istio.
+      // Minor and major upgrades most likely require manual adaptations of the code.
+      "matchDatasources": ["docker", "github-releases"],
+      "matchUpdateTypes": ["major", "minor"],
+      "matchPackagePatterns": [
+        "gcr\\.io\/istio-release\/.+",
+        "istio\/istio"
+      ],
```

```suggestion
      "matchPackagePatterns": [
        "europe-docker\\.pkg\\.dev\/gardener-project\/releases\/gardener\/autoscaler\/cluster-autoscaler",
        "gardener/autoscaler",
        "gcr\\.io\/istio-release\/.+",
        "istio\/istio"
      ],
```
We could restrict `cluster-autoscaler` to patch updates too, because there is an occurrence for each Kubernetes version ([ref](https://github.com/gardener/gardener/blob/8f03dfcaaddafe8b58e39592c907852e8f2cffac/imagevector/images.yaml#L120-L139)). Otherwise, there will be PRs which suggest to update all versions to the latest minor version.